lib/livebook/session.ex

```diff
@@ -2889,34 +2889,9 @@ defmodule Livebook.Session do
   # attributes that are missing.
   defp normalize_runtime_output(output)
 
-  defp normalize_runtime_output(%{type: :input, attrs: attrs} = output)
-       when attrs.type in [:text, :textarea, :password, :number, :url, :range, :color] and
-              not is_map_key(attrs, :debounce) do
-    default =
-      case attrs.type do
-        :range -> 250
-        _other -> :blur
-      end
-
-    put_in(output.attrs[:debounce], default)
-    |> normalize_runtime_output()
-  end
-
-  # Traverse composite outputs
-
-  defp normalize_runtime_output(output) when output.type in [:frame, :tabs, :grid] do
```

Comment:
I would at least keep this commented out, otherwise it's easy to forget that we need to do this recursively when adding future normalizations.